packages/api-page-builder/src/graphql/graphql/blocks.gql.ts

```diff
@@ -0,0 +1,50 @@
+import { GraphQLSchemaPlugin } from "@webiny/handler-graphql/types";
+import resolve from "./utils/resolve";
+import { PbContext } from "../types";
+
+export const createBlockGraphQL = (): GraphQLSchemaPlugin<PbContext> => {
+    return {
+        type: "graphql-schema",
```

Comment:
Instead of returning an object, we can return an instance of a new `GraphQlSchemaPlugin`. For example: `packages/api-admin-users-cognito/src/graphql/user.gql.ts:18`.

Reply:
@adrians5j 
have you missed this comment? or do nothing?

Reply:
I answered this via Slack @neatbyte-vnobis.

Sorry for the delay on this one, I actually missed your comment.